internal/pubsub/pubsub.go

```diff
@@ -179,11 +179,8 @@ func (s *Subscription) getNextBatch(ctx context.Context) error {
 			Body:     m.Body,
 			Metadata: m.Metadata,
 			ack: func() {
-				s.wg.Add(1)
-				go func() {
-					defer s.wg.Done()
-					s.ackBatcher.Add(ctx, ackIDBox{id})
-				}()
+				// TODO(ijt): Do something with the returned chan error.
```

Comment:
Based on our conversation yesterday, I think you can ignore the error. As long as you do the right thing after the call to SendAcks.

Reply:
Done.

What is "the right thing after the call to SendAcks"?